### PR TITLE
jpeg: bump revision and cleanup

### LINKED ISSUE #4355: jpeg: unit test failure
jpeg-9c-2 x86_64:
make check reports: Unsupported output file format for:
**djpeg -dct int -ppm -outfile testout.ppm ./testorig.jpg**

### PR DESCRIPTION
**Pull request by kenmays**
PASSES 100%. Fixed issue with unit testing.
Closes #4355

### CODE REVIEW

File: media-libs/jpeg/jpeg-9c.recipe [hunk 1]
```diff
@@ -11,13 +11,12 @@ on many machines ranging from PCs to Crays."
 HOMEPAGE="http://www.ijg.org/"
 COPYRIGHT="1991-2018 Thomas G. Lane, Guido Vollbeding"
 LICENSE="JPEG"
-REVISION="2"
+REVISION="3"
```
Comment (Begasus) [inline media-libs/jpeg/jpeg-9c.recipe]: If this is a bump version, reset to 1:
```diff
-REVISION="3"
+REVISION="1"
```

Comment (kenmays) [reply media-libs/jpeg/jpeg-9c.recipe]: Revision bump.

**Comment by korli** [comment]
There are unrelated changes in this PR. I'm inclined to close it as is.

**Comment by kenmays** [comment]
@korli - The revised recipe fixes the issue and I bumped the license. Review as needed.

**Comment by korli** [comment]
If you can't explain why it solves the issue, there is nothing to review.

**Comment by kenmays** [comment]
The issue is that jpegtrans and cjpeg/dpeg were not working properly due to some corruption. I removed the previous patch which seems to corrupt the build. Doing so, the tools work as expected. Also, the source license was outdated. Also, we don't use 9b - so cleanup was needed..

**Pull request merged**